### PR TITLE
temporary pinned apigateway to 0.7.0

### PR DESCRIPTION
**Pull request by csantanapr**

### CODE REVIEW

**Comment by csantanapr** [comment]
cc @mdeuser @rabbah

File: ansible/roles/apigateway/tasks/deploy.yml [hunk 1]
```diff
@@ -5,10 +5,11 @@
   shell: "docker pull openwhisk/apigateway"
   when: apigateway_local_build is undefined
 
+#temporary pinned to image 0.7.0 until apigateway fixes regressions
 - name: (re)start apigateway
   docker_container:
     name: apigateway
-    image: openwhisk/apigateway
+    image: openwhisk/apigateway:0.7.0
```
Comment (mdeuser) [inline ansible/roles/apigateway/tasks/deploy.yml]: Temp fix looks good to me

**Comment by rabbah** [comment]
Perhaps this should stay pinned. We can't keep getting bit by this.

**Comment by mdeuser** [comment]
Perhaps.... unfortunately, pinning in our build does not guarantee the version of the deployed api gw in ys1 or yp.

**Comment by rabbah** [comment]
Sigh. Good point. For the open source communit at least that not a concern and we should not break master and prs. For the deployed hosted version I don't see a compromise: we need to get the full test suite integrated, or decouple things further so that the impact is limited.

**Pull request merged**